systemFlake.nix

```diff
@@ -89,20 +89,24 @@ let
       selectedNixpkgs = getNixpkgs host;
       host = evalHostArgs (mergeAny hostDefaults host');
       patchedChannel = selectedNixpkgs.path;
+      channel = channels.${host.channelName} // { name = host.channelName; };
```

Comment:
I just want to pass channel metadata and passing the imported channel would result in a string which is very hard to use. Whereas input/name is more useful. Theres no place right now to edit all channels, so I think this is the best way to do it.